libs/langchain-google-common/src/types.ts

```diff
@@ -45,10 +57,59 @@ export interface GoogleConnectionParams<AuthOptions>
   platformType?: GooglePlatformType;
 }
 
+export enum GoogleAISafetyCategory {
```

Comment:
Ah we can't use enum for tree shaking reasons - can we just use a plain object?

Reply:
Will do!